addons/website_theme_install/static/src/less/website_theme_install.less

```diff
@@ -1,42 +1,164 @@
+@o-theme-kanban-gray: #fcfcfc;
+
 .o_kanban_view.o_theme_kanban {
+    background-color: @o-theme-kanban-gray;
+
+    /// Un-grouped Layout (default)
     &.o_kanban_ungrouped {
         .o-justify-content(center);
 
         .o_kanban_record {
-            .o-flex(0, 0, auto);
-            width: 100%;
-            @media (min-width: @screen-sm-min) {
-                width: 45%;
-            }
+            .o-flex(0, 1, auto);
+            width: 250px;
```

Comment:
It is fine like this but just to be sure: are you sure you want `.o-flex(0, 1, auto)` and not `.o-flex(1, 0, auto)` as I suggested ? Here you say "the records are always exactly 250px except when the viewport size is smaller than 250px" so in fact... the flex rule is nearly useless as I am not sure there are phones with that small resolution ;)

(in fact to be more correct, the rule is also useless as `flex: 0 1 auto` is the default for css ;) but it is not bad to put it anyway as some browser (IE) does not have the same default)